routerchain/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "routerchain",
+    "assets": [
+      {
+        "description": "The native token of Router Chain",
+        "extended_description": "Router Chain is a PoS L1 that allows stateful bridging–enabling truly omnichain dapps that can abstract chains from the users and aggregate liquidity across multiple chains.",
+        "denom_units": [
```

Comment:
The asset requires a base unit with 0 exponent.

Reply:
currently we have not set to any.

Reply:
Every asset must have a base denomination that represents indivisible units. If the on-chain unit is "route", then I'd recommend using "route" as exponent 0 (the base denom), and  "ROUTE" as exponent 18 (the display denom).

Reply:
To have this, we have to configure same on router chain also.

Reply:
Hey @JeremyParish69, we have added the denom.